bindings/java/build.gradle

```diff
@@ -0,0 +1,58 @@
+repositories {
+    google()
+    jcenter()
+}
+
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:4.0.0'
+    }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 29
+
+    defaultConfig {
+        minSdkVersion 28
+        targetSdkVersion 29
+
+        externalNativeBuild {
+            cmake {
+                // This is being checked in for now with the hard-coded linux
+                // paths due to some complications with configuring
+                // cross-compilation.
+                arguments '-DIREE_HOST_C_COMPILER=/usr/bin/clang',
+                        '-DIREE_HOST_CXX_COMPILER=/usr/bin/clang++',
+                        '-DIREE_BUILD_COMPILER=OFF',
+                        '-DIREE_ENABLE_MLIR=OFF',
+                        '-DIREE_BUILD_TESTS=OFF',
+                        '-DIREE_BUILD_SAMPLES=OFF',
+                        '-DIREE_BUILD_JAVA_BINDINGS=ON',
+                        '-DIREE_BUILD_PYTHON_BINDINGS=OFF',
+                        '-DIREE_HAL_DRIVERS_TO_BUILD=VMLA',
+                        '-DIREE_TARGET_BACKENDS_TO_BUILD=VMLA'
+            }
+        }
+    }
+
+    sourceSets {
+        main {
+            java.srcDirs = ['com/google/iree']
+            jni.srcDirs = ['com/google/iree/native']
+            manifest.srcFile '../javatests/com/google/iree/TestManifest.xml'
+        }
+    }
+
+    externalNativeBuild {
+        cmake {
+            path "../../CMakeLists.txt"
+        }
+    }
```

Comment:
Can this `path` be included in the prior `externalNativeBuild`? Or are two `externalNativeBuild` definitions needed?

Reply:
I believe this needs to be separate because the previous externalNativeBuild was in a defaultConfig block, and the CMake path needs to be specified outside of that